build.xml

```diff
@@ -541,8 +543,10 @@
 
           <dependency groupId="org.yaml" artifactId="snakeyaml" version="1.11"/>
           <dependency groupId="junit" artifactId="junit" version="4.12" />
+          <dependency groupId="org.mockito" artifactId="mockito-core" version="3.2.4" />
```

Comment:
this is already on trunk, is something up with the rebase?

Reply:
doesn't show up in the latest rebase